IMILA/Messenger.Persistence/Entities/UserAccount.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Messenger.Persistence.Entities.Common;
+
+namespace Messenger.Persistence.Entities
+{
+    internal class UserAccount : IAuditable
+    {
+        public long Id { get; set; }
+        public long UserId { get; set; }
+        public required string PasswordHash { get; set; }
+        public required string PasswordSalt { get; set; }
+        public required bool IsActive { get; set; }
```

Comment:
value type

Reply:
cia del required? kaip ir sutarem pabandyt palikt ji?

Reply:
na, jeigu tikrai braukia, tai okay. Bet that is strange

Reply:
mes  man value typu nebraukia

Reply:
![image](https://github.com/Erikas/Messenger/assets/5561591/cd924a9f-5d8b-40cf-b96a-1c5d7b151dca)

Reply:
Sita idejus nebebraukia: <Nullable>disable</Nullable> nezinau kiek gerai :D

Reply:
<Nullable>disable</Nullable>